plugins/3rdParty/qRDBIO/CMakeLists.txt

```diff
@@ -4,8 +4,11 @@ option( PLUGIN_IO_QRDB "Check to install RDB 2 I/O plugin" OFF )
 
 # CloudCompare RDB 2 I/O plugin
 if (PLUGIN_IO_QRDB)
+
 	project( QRDB_IO_PLUGIN )
 
+	AddPlugin( NAME ${PROJECT_NAME} )
```

Comment:
This will need to identify the plugin type which is used for installation of the plugin.

```cmake
AddPlugin( NAME ${PROJECT_NAME} TYPE io )
```